### ISSUE TITLE
HTTP-date vs. rfc1123-date (Trac #51)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#51]
Should rfc1123-date BNF be used where the specification calls for something to be a HTTP-date, but only the RFC1123 form? Should separate rules for producing and consuming dates be specified?

Migrated from https://trac.ietf.org/ticket/51
```json
{
    "status": "closed",
    "changetime": "2008-04-09T13:45:42",
    "_ts": "1207748742000000",
    "description": "Should rfc1123-date BNF be used where the specification calls for something to be a HTTP-date, but only the RFC1123 form? Should separate rules for producing and consuming dates be specified?",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2007-12-22T02:33:40",
    "component": "p1-messaging",
    "summary": "HTTP-date vs. rfc1123-date",
    "priority": "",
    "keywords": "",
    "milestone": "unassigned",
    "owner": "",
    "type": "editorial",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20080104T05:59:38: 
* mnot@pobox.com changed component from "" to "auth"
* mnot@pobox.com changed milestone from "" to "unassigned"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080104T06:22:54: 
* mnot@pobox.com changed component from "auth" to "messaging"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080403T01:05:06: mnot@pobox.com changed type from "design" to "editorial"

**Comment by mnot** [comment]
Trac update at 20080409T13:45:42: 
* julian.reschke@gmx.de commented:

> Fixed in [239]:
> 
> Resolve #51: make it obvious from the BNF that most of the date formats that must be accepted are indeed obsolete (closes #51).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"
